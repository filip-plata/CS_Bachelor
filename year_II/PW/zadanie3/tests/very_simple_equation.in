5 1 3
1 x[0] = x[1]
2 x[1] -1000
3 x[1] 5
4 x[1] 966
5 x[0] 12
