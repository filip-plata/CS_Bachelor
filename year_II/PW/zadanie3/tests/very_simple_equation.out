1 P
2 P -1000
3 P 5
4 P 966
5 P 12
